cleanup.yml

```diff
@@ -4,26 +4,32 @@
         file:
             path: "{{ infrared_dir }}"
             state: absent
-      - name: remove {{ infrared_workspaces_dir }}
+
+      - name: remove .infrared dir
         file:
-            path: "{{ infrared_workspaces_dir }}"
+            path: "{{ ansible_user_dir }}/.infrared"
```

Comment:
I am seeing these changes in another PR also. i think it got missed while separating the works.

Reply:
Done